docker/systemctl-dummy

```diff
@@ -0,0 +1,2 @@
+!/bin/bash
```

Comment:
If this is a bash script, use an appropriate shebang.

```suggestion
#!/bin/bash
```

Reply:
Thanks, I will fix typo.